src/jco.js

```diff
@@ -81,6 +81,18 @@ program.command('types')
   .option('--all-features', 'enable all features')
   .action(asyncAction(types));
 
+program.command('guest-types')
+  .description('Generate guest types for the given WIT')
```

Comment:
Perhaps let's note that this is experimental for now?

```suggestion
  .description('Generate experimental guest types for the given WIT')
```